index.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>UN Climate Change</title>
+  </head>
+
+  <body>
+    <!--  header -->
+    <header>
+      <ul class="header1">
+        <li>
+          <a href="#"><img src="./img/facebook.png" alt="facebook logo"/></a>
+        </li>
+        <li>
+          <a href="#"><img src="./img/twitter.png" alt="twitter logo"/></a>
+        </li>
+        <li class="header1-text">English</li>
+        <li class="header1-text">My Page</li>
+        <li class="header1-text">Logout</li>
+      </ul>
+
+      <div class="nav-bar">
+        <ul class="humberger">
+          <li class="bar"></li>
+          <li class="bar"></li>
+          <li class="bar"></li>
+        </ul>
+        <ul class="nav-items">
+          <li>
+            <a href="#" class="page1">
+              <img class="logo" src="./img/logo.png" alt="logo image"
+            /></a>
+          </li>
+          <li class="header-text1"><a href="#" class="page1">Home</a></li>
+          <li><a href="#" class="about-page">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#partner">Sponsor</a></li>
+          <li class="logo2">UN climate change</li>
+        </ul>
+      </div>
+    </header>
+
+    <!-- page 1-->
+    <div class="page1">
+      <section class="home">
+        <div class="home-title">
+        <h3 class="sub-title">Sharm el-Sheikh Climate Change Conference - November 2022</h3>
+        <h1 class="main-title">
+          What did COP 27 achieve and what needs to happen next?
+        </h1>
+        <p class="paragraph1">
+          Establishing a dedicated fund for loss and damage,
+          Maintaining a clear intention to keep 1.5°C within reach, Holding businesses and institutions to account, 
+          Mobilizing more financial support for developing countries, and Making the pivot toward implementation.
+        </p>
```

Comment:
[REUIRED] Kindly align the texts to the left to match the template

[REQUIRED] Also ensure the text colour matches the template. One of the aims of this project is to train to follow client requirements. Kindly follow the design in terms of color, spacing and typography.

![image](https://user-images.githubusercontent.com/62938735/212382835-0dd60975-0af8-47fe-a8a7-7ed018efabd0.png)